code/datums/components/chameleon/_chameleon.dm

```diff
@@ -0,0 +1,163 @@
+/datum/component/chameleon
+	/// The original name for this chameleon item, such as "chameleon jumpsuit"
+	var/original_name = "Object"
+	/// The base type that disguises will be fetched from.
+	var/base_disguise_path
+	/// A typecache of items not allowed to be selected for this item.
+	var/list/disguise_blacklist
+	/// A normal list of paths of which disguises can be selected.
+	var/list/disguise_paths = list()
+	/// An associative list (name = callback) of "extra actions" to present in the UI for this item.
+	var/list/extra_actions = list()
+	/// Whether anyone is allowed to use this chameleon item, or just specific people.
+	var/anyone_can_use = TRUE
```

Comment:
```suggestion
	var/anyone_can_use = FALSE
```
Just reminding myself to ensure I change this back to `FALSE` before I merge it.
Not right now though, having it `TRUE` is good for testing.